server/spike/adk/spike_12_multi_agent_validation.py

```
@@ -0,0 +1,525 @@
+    )
+
+    result = await run_agent(orchestrator, "Research cloud computing market size. Be very brief.")
+
+    # Check event attribution
+    events_by_author: dict[str, list[str]] = {}
+    for ev in result["events"]:
+        author = ev["author"]
+        for part in ev["parts"]:
+            events_by_author.setdefault(author, []).append(part["type"])
+
+    print(f"  Events by author:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)